_posts/2015-01-26-monster-fighter.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Monster fighter
+author: pvto
+excerpt: A developer's guide for surviving a large and long running project.
+---
+
+Working with **a large** and **long running** project can be devastating.  More so, if in production and striving to maintain **a good service level** while at the same time **continuously shipping new features**.  As a side effect, you see **technologies aging** and you may start to worry about your professional competence in the wild.
```

Comment:
"if in production and striving to maintain" -> "if it's in production and you're striving to maintain"